packages/cobol-ls/package.yaml

```diff
@@ -0,0 +1,29 @@
+---
+name: cobol-ls
+description: COBOL Language Support provides autocomplete, highlighting and diagnostics for COBOL code and copybooks
+homepage: https://github.com/eclipse-che4z/che-che4z-lsp-for-cobol
+licenses:
+  - EPL-2.0
+languages:
+  - COBOL
+categories:
+  - LSP
+
+source:
+  id: pkg:github/eclipse-che4z/che-che4z-lsp-for-cobol@2.1.1
+  asset:
+    - target: darwin_arm64
+      file: cobol-language-support-darwin-arm64-{{version}}.vsix
+    - target: darwin_x64
+      file: cobol-language-support-darwin-x64-{{version}}.vsix
+    - target: linux_arm64
+      file: cobol-language-support-linux-arm64-{{version}}.vsix
+    - target: linux_x64
+      file: cobol-language-support-linux-x64-{{version}}.vsix
+    - target: win_arm64
+      file: cobol-language-support-win32-arm64-{{version}}-signed.vsix
+    - target: win_x64
+      file: cobol-language-support-win32-x64-{{version}}-signed.vsix
+
+bin:
+  cobol-language-support: "{{source.asset.bin}}"
```

Comment:
`{{source.asset.bin}}` gets expanded to `nil` because it's not defined above, resulting in no bin being linked. Do they have a recommended way of running the server? It seems like the assets contain compiled Java bytecode as well as a platform-specific GraalVM binary

Reply:
It also has a native binary found in extension/server/native. Maybe I should add that path to the source